### PR TITLE
[16.0][ADD] purchase_order_hide_receipt_status

### PR DESCRIPTION
**Pull request by rinaldifirdaus**
This module hides receipt_status field from purchase order form view and tree view to avoid any confusion that may arise when 'reception_status' field becomes visible once the 'purchase_reception_status' module is installed.

@qrtl

### CODE REVIEW

File: purchase_order_hide_receipt_status/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Copyright 2023 Quartile Limited
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+{
+    "name": "Purchase Order Hide Receipt Status",
+    "version": "16.0.1.0.0",
+    "category": "Purchase",
+    "website": "https://github.com/OCA/purchase-workflow",
+    "license": "AGPL-3",
+    "author": "Quartile Limited, Odoo Community Association (OCA)",
+    "depends": ["purchase_stock", "purchase_reception_status"],
```
Comment (yostashiro) [inline purchase_order_hide_receipt_status/__manifest__.py]: `purchase_reception_status` is not a technical dependency, so let's remove it.

```suggestion
    "depends": ["purchase_stock"],
```

File: purchase_order_hide_receipt_status/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+    "data": ["views/purchase_order.xml"],
+    "installable": True,
+    "auto_install": True,
```
Comment (yostashiro) [inline purchase_order_hide_receipt_status/__manifest__.py]:
```suggestion
```

**Review by yostashiro — CHANGES_REQUESTED** [review]

**Comment by yostashiro** [comment]
@rinaldifirdaus Please squash commits.

**Comment by rinaldifirdaus** [comment]
> @rinaldifirdaus Please squash commits.

Done, Thank you for your review!

**Review by yostashiro — APPROVED** [review]

**Review by AungKoKoLin1997 — APPROVED** [review]

**Pull request merged**